### PR TITLE
handle additional card action types

### PR DESCRIPTION
**Pull request by eanders-MS**

### CODE REVIEW

**Comment by eanders-MS** [comment]
For issue https://github.com/Microsoft/BotFramework-WebChat/issues/271

**Comment by eanders-MS** [comment]
My editor likes to automatically trim trailing whitespace.

File: src/History.tsx [hunk 1]
```diff
@@ -64,7 +64,12 @@ class HistoryContainer extends React.Component<Props, {}> {
...
                 break;
 
+            case "call":
             case "openUrl":
+            case "playAudio":
+            case "playVideo":
+            case "showImage":
+            case "downloadFile":
             case "signin":
                 window.open(value);
```
Comment (danmarshall) [inline src/History.tsx]: Hmmm - this doesnt open in a new tab. It will open in the current window.

Comment (eanders-MS) [inline src/History.tsx]: When I tested locally all opened in a new tab.

Comment (danmarshall) [inline src/History.tsx]: Could be true for the mime types that don't resolve to ```text/*``` 
But for ```case "openUrl":``` and ```case "signin":``` these might display in the current window if they are html

**Comment by eanders-MS** [comment]
According to window.open docs, it always opens in a new window (or tab, I suppose):
http://www.w3schools.com/jsref/met_win_open.asp
"The open() method opens a new browser window."

File: src/History.tsx (hunk 1 above)
Comment (eanders-MS) [inline src/History.tsx]: According to window.open docs, it always opens in a new window (or tab, I suppose):
http://www.w3schools.com/jsref/met_win_open.asp
"The open() method opens a new browser window.

Comment (danmarshall) [inline src/History.tsx]: oic - ```_blank``` is default. tx for checking :)

**Pull request merged**